### DISCUSSION TITLE
Volunteer- Meeting Minutes 06/2/2022

### DISCUSSION THREAD

**Discussion by yangli-os** [WeBankFinTech/WeDataSphere#33] [General]
Date 日期：05月/ 19 日 /2022年
Place: online

Members in attendance 出席人员

@brucedchen1991 [Peace chen](https://github.com/orgs/WeBankFinTech/people/brucedchen1991)
@Ritakang0451[康月牙ritakang](https://github.com/orgs/WeBankFinTech/people/Ritakang0451)
@yangli-os[YangLi](https://github.com/orgs/WeBankFinTech/people/yangli-os)
@Clairedenggg[Yongjia Teng](https://github.com/orgs/WeBankFinTech/people/Clairedenggg)
@Yuhang-Luky[Yuhang-Luky](https://github.com/orgs/WeBankFinTech/people/Yuhang-Luky)

Scheduled  time 会议时间: 20:00-21:00

Discussion topic 讨论话题:
会议记录：
REVIEW
1. @peace提交了 Apache linkis网站 download模块的PR
2. @luck宇航开始做WeDataSphere内容在知乎的转载
3. @李扬优化了apache email[notic] 翻译sop
4. @rita增加了哔哩哔哩的SOP
5.看板仍旧使用GitHub自带的看板功能。

Bullet points 重点记录
1.对比其它apache项目网站做网站的内容优化；可以自己整改先提交issue，然后提交PR，（例如：peace更新的download模块。同时，可以录制/写一下，如何参与apache贡献，比如标题：看这个视频/文章，10min教你走出新手村，成为apache贡献者。@rita@永嘉，录制和剪辑视频。）@ALL
2.图床模块的使用，跟linkis贡献关联，并出一个教程@李扬
3.调研使用issues pr来更显性的分配任务，去看一下wuhan2020社区或者其它社区的做法@李扬
3.规范化处理issues的任务（暂时挂起）
4.增加看板当中需要翻译的文章链接，大家认领翻译任务自己在后面@自己 ，@rita@李扬
6.将微信公众号文章转载至知乎@宇航，转载内容的翻译@永嘉
7.录制linkis安装视频的相关问题，@rita跟进@john wong。（如系统的最低要求，集群的咨询先在社区用户群中解决基础问题，或者在开发群中和给各位沟通，另外可以请教casinoxia）

其他
关于开源项目的组织结构&角色情况：
https://github.com/Ritakang0451/OPEN-SOURCE-GOVERNANCE-SYSTEM/blob/main/Operating%20Manual/How%20to%20Get%20involved%20in%20Open%20Source/Organizational%20System%20%26%20Role%20Definition.md
+李扬的开源通史的思维导图

Date of next meeting下次会议时间: 每两周，6月第三周,下次会议主持@邓永嘉

**Comment by Yuhang-Luky** [comment]
感谢您的来信，小卢同学已收到！

**Comment by linzhenx** [comment]
您的邮件已经顺利到达我的邮箱中了~  谢谢您的邮件

**Comment by Clairedenggg** [comment]
好的👌收到🫡(´▽｀)

On Jun 3, 2022, at 22:12, YangLi ***@***.***> wrote:

﻿

Date 日期：05月/ 19 日 /2022年
Place: online

Members in attendance 出席人员

@brucedchen1991<https://github.com/brucedchen1991> Peace chen<https://github.com/orgs/WeBankFinTech/people/brucedchen1991>
@Ritakang0451<https://github.com/Ritakang0451>康月牙ritakang<https://github.com/orgs/WeBankFinTech/people/Ritakang0451>
@yangli-os<https://github.com/yangli-os>YangLi<https://github.com/orgs/WeBankFinTech/people/yangli-os>
@Clairedenggg<https://github.com/Clairedenggg>Yongjia Teng<https://github.com/orgs/WeBankFinTech/people/Clairedenggg>
@Yuhang-Luky<https://github.com/Yuhang-Luky>Yuhang-Luky<https://github.com/orgs/WeBankFinTech/people/Yuhang-Luky>

Scheduled time 会议时间: 20:00-21:00

Discussion topic 讨论话题:
会议记录：
REVIEW

  1.  @peace<https://github.com/peace>提交了 Apache linkis网站 download模块的PR
  2.  @luck<https://github.com/luck>宇航开始做WeDataSphere内容在知乎的转载
  3.  @李扬优化了apache email[notic] 翻译sop
  4.  @rita<https://github.com/rita>增加了哔哩哔哩的SOP
5.看板仍旧使用GitHub自带的看板功能。

Bullet points 重点记录
***@***.***<https://github.com/rita>@***@***.***<https://github.com/ALL>
2.图床模块的使用，跟linkis贡献关联，并出一个教程@李扬
3.调研使用issues pr来更显性的分配任务，去看一下wuhan2020社区或者其它社区的做法@李扬
3.规范化处理issues的任务（暂时挂起）
4.增加看板当中需要翻译的文章链接，大家认领翻译任务自己在后面@自己 ***@***.***<https://github.com/rita>@李扬
6.将微信公众号文章转载至知乎@宇航，转载内容的翻译@永嘉
***@***.******@***.***<https://github.com/john> wong。（如系统的最低要求，集群的咨询先在社区用户群中解决基础问题，或者在开发群中和给各位沟通，另外可以请教casinoxia）

其他
关于开源项目的组织结构&角色情况：
https://github.com/Ritakang0451/OPEN-SOURCE-GOVERNANCE-SYSTEM/blob/main/Operating%20Manual/How%20to%20Get%20involved%20in%20Open%20Source/Organizational%20System%20%26%20Role%20Definition.md
+李扬的开源通史的思维导图

Date of next meeting下次会议时间: 每两周，6月第三周,下次会议主持@邓永嘉
